src/main/java/org/broadinstitute/hellbender/tools/reference/CompareReferences.java

```diff
@@ -204,6 +256,193 @@ private void displayBySequenceName(Set<ReferenceSequenceTable.TableRow> rows, St
+                        int pos = Integer.valueOf(fields[0]);
+                        String ref = fields[1];
+                        String alt = fields[2];
+
+                        // insertion
+                        if (ref.equals(".") && !alt.equals(".")) {
+                            if (pos == previousPos && currentIndel.isInsertion) {
+                                currentIndel.alt += alt;
+                            } else {
+                                if (currentIndel != null) {
+                                    writer.add(currentIndel.getAsVCFRecord());
+                                }
+                                String refPadding = new String(source.queryAndPrefetch(new SimpleInterval(contig, pos-1, pos-1)).getBases());
+                                currentIndel = new MummerIndel(contig, refPadding, refPadding + alt, pos-1, true, false);
+                            }
+                        }
+
+                        // deletion
+                        else if(!ref.equals(".") && alt.equals(".")){
+                            if(pos == previousPos+1 && currentIndel.isDeletion){
+                               currentIndel.ref += ref;
+                            }
+                            else {
+                                if(currentIndel != null) {
+                                    writer.add(currentIndel.getAsVCFRecord());
+                                }
+                                String refPadding = new String(source.queryAndPrefetch(new SimpleInterval(contig, pos-1, pos-1)).getBases());
+                                currentIndel = new MummerIndel(contig, refPadding + ref, refPadding, pos-1, false, true);
+                            }
+                        }
+                        // snp
+                        else {
+                            if(currentIndel != null){
+                                writer.add(currentIndel.getAsVCFRecord());
+                                currentIndel = null;
+                            }
+                            VariantContextBuilder vcfBuilder = new VariantContextBuilder();
+                            VariantContext record = vcfBuilder.chr(contig).start(pos).stop(pos).alleles(ref, alt).make();
+                            writer.add(record);
+                        }
+                        previousPos = pos;
+                    }
+                    if(currentIndel != null){
+                        writer.add(currentIndel.getAsVCFRecord());
+                    }
+            }
+            catch(IOException e){
+                throw new UserException("");
+            }
+            catch(NumberFormatException e){
+                throw new UserException("");
+            }
+        }
+        else{
+            logger.info("No mismatching sequences found.");
+        }
+    }
+
+    private void runFindSNPS(ReferencePair refPair, ReferenceSequenceTable table) {
```

Comment:
It came up in the discussion, you may want to add an "ignore capitalization" mode and argument here.

Reply:
If not we should make it a new issue in gatk so we track it later